hack/build/common.sh

```diff
@@ -44,5 +44,6 @@ VENDOR_DIR=${WASP_DIR}/vendor
 ARCHITECTURE="${BUILD_ARCH:-$(uname -m)}"
 HOST_ARCHITECTURE="$(uname -m)"
 WASP_CRI="$(determine_wasp_bin)"
+KUBEVIRT_RELEASE=${KUBEVIRT_RELEASE:-"latest_nightly"}
```

Comment:
WDYT about changing this the latest_stable i think that it's good enough since wasp is not coupled with kubevirt.